bungeecord/build.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-    compileOnly group: "net.md-5", name: "bungeecord-api", version: "1.8-SNAPSHOT"
+    compileOnly group: "net.md-5", name: "bungeecord-api", version: "1.16-R0.4-SNAPSHOT"
```

Comment:
See above - what does changing this enable us? We don't need new MC versions.

Reply:
A normal update because time is moving on and everyone hates outdated Minecraft versions.

Reply:
No, this just introduces the risk that someone might depend on a newer API version and break older versions. Revert this.

Reply:
(Granted, this is probably more justifiable as almost everyone upgrades their BungeeCord versions.)